web/src/layout/Header/navbar/Menu/Settings/SendMeNotifications/FormNotifs/FormEmail.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { Dispatch, SetStateAction, useEffect } from "react";
+import styled from "styled-components";
+
+const StyledInput = styled.input`
```

Comment:
Check if the component from the UI package could be used here, as it has ways to add error messages